rollup.config.js

```diff
@@ -1,45 +1,72 @@
-import babel from '@rollup/plugin-babel';
+import { getBabelOutputPlugin } from '@rollup/plugin-babel';
 const commonjs = require('@rollup/plugin-commonjs');
 const { nodeResolve } = require('@rollup/plugin-node-resolve');
 const nodePolyfills = require('rollup-plugin-node-polyfills');
+import typescript from '@rollup/plugin-typescript';
 
 // Entry point of the library
-const input = './lib/bson.js';
+const input = './src/bson.ts';
 
 export default [
+  {
+    input,
+    /* Browser ESM bundle */
```

Comment:
can you explain what's going on in this change? Also, it looks like the config is mostly duplicated below

Reply:
Yea the key difference is the babel settings. There is definitely a lot of duplication but after wrestling with the config for a better part of this project an explicit set of settings for each output is actually quite helpful rather than debugging a programatic config.